mods/sethome/init.lua

```diff
@@ -27,7 +35,7 @@ loadhomes()
 sethome.set = function(name, pos)
 	local player = minetest.get_player_by_name(name)
 	if not player or not pos then
-		return false
+		minetest.chat_send_player(player:get_player_name(), "Player not found!")
```

Comment:
Haha, this is ridiculous.

Just return the `bool` (mods might use this function, too).

Reply:
But when I was returning it, it would send the message as soon as the warning poped up. I'd love to simplify it but I'm not proficient enough in Lua.